OgreMain/src/OgreTextureUnitState.cpp

```diff
@@ -292,7 +302,7 @@ namespace Ogre {
         if (ct == CONTENT_SHADOW || ct == CONTENT_COMPOSITOR)
         {
             // Clear out texture frames, not applicable
-            mFrames.clear();
+            setNumFrames(0);
```

Comment:
even here `mFrames.size() == mNumFrames`, so I still think we can drop `mNumFrames`

Reply:
I am trying to just remove mFrames, as it duplicates information with mFramePtrs.

Reply:
needs more thought, see #677